src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -985,6 +985,36 @@ def test_set_privacy_nomsg(self):
         self.get_addon().update(eula=None, privacy_policy=None)
         self.is_success(self.get_dict(has_priv=True))
 
+    def test_source_submission_notes_not_shown_by_default(self):
+        url = reverse('devhub.submit.source', args=[self.addon.slug])
+        response = self.client.post(url, {
+            'has_source': 'no',
+            'source': None,
+        }, follow=True)
```

Comment:
You could have just set the submitted source property on the version (or wherever it's set) but this works too.